docker/windows/Dockerfile-houdini

```diff
@@ -0,0 +1,18 @@
+# escape=`
+
+ARG BASE_IMAGE="palladio-tc-base:win19-vc1437-v0"
+FROM ${BASE_IMAGE}
+SHELL ["cmd", "/S", "/C"]
+
+ARG HOUDINI_VERSION=20.0.751
```

Comment:
Change the default to 20.5 after rebasing (cf. PR #243)?

Reply:
done